sounds/tests/test_manager.py

```diff
@@ -55,11 +43,11 @@ def setUp(self):
 
     def test_bulk_query_id_num_queries(self):
 
-        # Check that all fields for each sound are retrieved with one query
-        with self.assertNumQueries(1):
+        # Check that all fields for each sound are retrieved with one query + one for the analyzers
+        with self.assertNumQueries(2):
             for sound in Sound.objects.bulk_query_id(sound_ids=self.sound_ids, include_analyzers_output=True):
                 for field in self.fields_to_check_bulk_query_id:
```

Comment:
This test is now missing a bit now that we've removed all of the "extra fields". There's no way to check that accessing a related item will not perform another query. We should extend it with a list of foreign keys and many-to-many relations that we should access to ensure that we stay with the same number of queries